azure-pipelines.yml

```diff
@@ -41,7 +41,7 @@ stages:
   jobs:
   - job: 'Linux'
     pool:
-      vmImage: 'ubuntu-16.04'
+      vmImage: 'ubuntu-20.04'
```

Comment:
How about just using the `ubuntu-latest`? What's the downside?

Reply:
@DusKing1 discused here:
https://github.com/betaflight/betaflight-configurator/pull/2505/files/6aaf63656d99e03759f8191cfd6828fe52d5f529#r646302183

Reply:
K, I got that.